Update README.md

sfarhan5000 opened the pull request.

sfarhan5000 committed:
Update README.md

bytemaster-0xff commented:
VERY COOL!!!!

Learning GITHUB and especially pull requests will really set you guys apart and looks awesome on your resumes 😊

Kevin…

From: sfarhan5000 <notifications@github.com>
Sent: Friday, January 15, 2021 13:57
To: USFCapstone-I/smallfarm <smallfarm@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: [USFCapstone-I/smallfarm] Update README.md (#1)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/USFCapstone-I/smallfarm/pull/1

Commit Summary

  *   Update README.md

File Changes

  *   M README.md<https://github.com/USFCapstone-I/smallfarm/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5> (2)

Patch Links:

  *   https://github.com/USFCapstone-I/smallfarm/pull/1.patch
  *   https://github.com/USFCapstone-I/smallfarm/pull/1.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/USFCapstone-I/smallfarm/pull/1>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABHAFTE52Z525DUZUE3QY33S2CFZRANCNFSM4WEPAQQQ>.

sfarhan5000 committed:
Quick update

Trying to write to the main  branch using visual studio code editing and then commiting to github main file.

sfarhan5000 committed:
Update README.md

More practice on pushing commits!
